bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/PresenceDetectionValue.java

```diff
@@ -27,131 +30,124 @@
  */
 @NonNullByDefault
 public class PresenceDetectionValue {
-    private double latency;
-    private boolean detectionIsFinished;
-    private final Set<PresenceDetectionType> reachableByType = new TreeSet<>();
-    private final List<Integer> tcpServiceReachable = new ArrayList<>();
-    private final String hostAddress;
 
-    /**
-     * Returns true if the target is reachable by any means.
-     */
-    public boolean isReachable() {
-        return latency >= 0;
-    }
+    public static final Duration UNREACHABLE = Duration.ofMillis(1).negated();
```

Comment:
Can it be simplified to:
```suggestion
    public static final Duration UNREACHABLE = Duration.ofMillis(-1);
```

?

Reply:
That will also work just fine.